dfx.json

```
@@ -28,11 +28,21 @@
       "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm.gz",
       "remote": {
         "id": {
-          "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
+          "ic": "mxzaz-hqaaa-aaaar-qaada-cai"
```

```suggestion
          "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
```

typo, this should have stayed like it was